### ISSUE TITLE
Global Configuration, Commit Message Regex, and Author Regex

### ISSUE THREAD

**Issue by gabriellebautista** [tomasbjerre/simple-bitbucket-commit-checker#57]
New Features:
Global Configuration (so we can enable this hook on all the repos).
Commit Message Regex so we can validate all the commit messages
Author Regex so we can ignore service accounts which automates some commits

**Comment by tomasbjerre** [comment]
Global configuration can be done with: https://github.com/tomasbjerre/settings-synchronizer-for-bitbucket-plugin

Commit message regexp is at the bottom of the configuration GUI. The rules/groups solves this.

There is a checkbox to ignore service users. There is also a regexp on committer email that you may use.

**Comment by gabriellebautista** [comment]
Cool! We've tried everything you suggested and works like a charm! We have also opt to use "Ignore Users" for service accounts.

Follow up questions:
When we use the commit message regex validation, the regex used is being returned to the user but we already configured a 'friendly message' for this one. Can we exclude the regex?
Any known performance issues? We have around 6000 git repositories and 7000 accounts
